### ISSUE TITLE
I couldn't find the details where we can find the data_collection_rule_id and stream_name after creating the DCR for Custom Log 

### ISSUE THREAD

**Issue by anishsingh0302** [Azure/azure-sdk-for-net#42584]
### Type of issue

Missing information

### Description

Hi, 
I have not found the details of where we can find the data collection rule id and Stream name after creating the DCR for ingestion of log through api. could you please here to update the document or you can help here as well to where we can find the details.

### Page URL

https://learn.microsoft.com/en-us/dotnet/api/overview/azure/monitor.ingestion-readme?view=azure-dotnet

### Content source URL

https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/api/overview/azure/latest/monitor.ingestion-readme.md

### Document Version Independent Id

5da798d5-e439-c614-a39f-b57baf15207d

### Article author

@azure-sdk

### Metadata

* ID: f40b578b-5251-044a-c16d-050d29b318dd 
* Service: **monitor**

**Comment by jsquire** [comment]
Thank you for your feedback.  Tagging and routing to the team member best able to assist.

**Comment by anishsingh0302** [comment]
@nisha-bhatia  could you please help here to find data collection rule ID?

**Comment by nisha-bhatia** [comment]
Hi @anishsingh0302, this link has information about how to view details in the DCR: https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/data-collection-rule-overview?tabs=portal please let me know if you can't find what you need in here. I'll add it to the Ingestion docs as well.

**Comment by anishsingh0302** [comment]
Yes, I could not find here what will be data collection rule id and Stream name after creations of DCR here. you can specified or provide description document that Once DCR is created, we can find the details of these value on opening of view json file On DCR overview.
